### ISSUE TITLE
weight in loss computation

### ISSUE THREAD

**Issue by zjhthu** [princeton-vl/SEA-RAFT#5]
In the paper, the loss weight for two terms are $\alpha \in [0, 1]$ and   $1 - \alpha $, but in the code, the network predicts two weights rather than one, and the loss is in the form `logsumexp(weight) - logsumexp(weight - loss)`, why?

```
                raw_b = info_predictions[i][:, 2:]
                log_b = torch.zeros_like(raw_b)
                weight = info_predictions[i][:, :2]
                # Large b Component                
                log_b[:, 0] = torch.clamp(raw_b[:, 0], min=0, max=var_max)
                # Small b Component
                log_b[:, 1] = torch.clamp(raw_b[:, 1], min=var_min, max=0)
                # term2: [N, 2, m, H, W]
                term2 = ((flow_gt - flow_predictions[i]).abs().unsqueeze(2)) * (torch.exp(-log_b).unsqueeze(1))
                # term1: [N, m, H, W]
                term1 = weight - math.log(2) - log_b
                nf_loss = torch.logsumexp(weight, dim=1, keepdim=True) - torch.logsumexp(term1.unsqueeze(1) - term2, dim=2)
```

**Comment by MemorySlices** [comment]
We use softmax to obtain the probabilities, similar to image classification.

**Comment by NEUzhl** [comment]
?I can not understand

**Comment by yanqswhu** [comment]
@zjhthu @NEUzhl you can check my issue to understand these codes #22
